edx/xblock-lti-consumer
# [BD-24] [TNL-7318] BB-2355: Move LTI configuration to models.

This PR moves the LTI configuration handling to a django model. All the settings are still stored in the XBlock, but the retrieving and instancing of LTI config is done through a Python API now.

**Testing instructions:**
1. Checkout and install this branch.
2. Set up a LTI 1.1 and a LTI 1.3 blocks in a course (follow the guide on https://github.com/edx/xblock-lti-consumer#testing-against-an-lti-provider).
3. Test that LTI 1.1 works from the LMS and Studio.
4. Test that LTI 1.3 launch works from LMS.
5. Check on the LTI Configuration on Django admin.

**Notes:**
There's no new "features" here, and this is just laying the groundwork to support LTI Advantage extensions.
This is the first implementation step of https://github.com/edx/xblock-lti-consumer/pull/92.

**Reviewers:**
